challenge.md

```diff
@@ -32,19 +32,20 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
- 
+ It's an absolute path because it specifies several sub directories
+
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
- 
+ Absolute path, because you its not in a location in your current working directory
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
- 
+Relative is fine because its within the current directory, which is where the computer looks if the path is relative.
```

Comment:
Sort of. We want to use a Relative path so that if we zip up our project and give it to someone else, all the links won't break when they open it on their computers.